.github/workflows/eslint.yml

```diff
@@ -0,0 +1,27 @@
+name: eslint
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Read .nvmrc and pass it on
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+
+      - name: Setup node w/ nvm version
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+      - name: yarn install w/ cache
+        uses: bahmutov/npm-install@v1
+
+      - name: ESlint
+        run: yarn eslint
```

Comment:
It's hard to argue that the 30 seconds each one of these takes is "slower" when jest will be our bottle neck. 

I am curious about the amount of duplicate config. It's not _that_ big of a deal, but it's certainly a PITA. 

I looked to see if you could do the thing that I was talking about and in one job install the things, then split into 3 to do testing, linting, typing, etc, seems no.

SO, I'm curious about a few things.

- Why multiple workflows instead of one with multiple jobs? Why not a "Pull request" workflow with 3 jobs in there.
- It looks like authoring an action is also a bit of a hassle (can't just define a collection of steps and call it an action), thoughts about how we might get this "Setup and install node with a cache" step so it's only defined in one place? It's probably not worth fixing now as it wont change often, but having it in three places is a bit wonky.
    - We can create the action in our own repo and just reference it, but it's got to be node or docker it seems.

Reply:
Discussed offline.

- Seems faster as three workflows vs 3 jobs in one workflow, will investigate today
- Duplication does suck, also was going to investigate this today as well.